### PR TITLE
added sort functions

### PR DESCRIPTION
**Pull request by mingoedjudah**
added sort functions for mingo_edjudah folder

### CODE REVIEW

**Review by fidel-dianarose — APPROVED** [review]
LGTM!

File: mingo_edjudah/styles.css [hunk 1]
```diff
@@ -185,3 +185,4 @@ body {
         width: calc(100% - 16px);
     }
 }
+
```
Comment (nabayra-james) [inline mingo_edjudah/styles.css]: remove this line

**Review by nabayra-james — CHANGES_REQUESTED** [review]
remove unnecessary line

File: mingo_edjudah/styles.css (hunk 1 above)
Comment (mingoedjudah) [reply mingo_edjudah/styles.css]: yuh

**Review by nabayra-james — APPROVED** [review]
LGTM!

**Pull request merged**